frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/components/uploadListItemRow/uploadListItemRow.styles.ts

```diff
@@ -41,6 +40,9 @@ export const UploadListItem = styled(DashboardListItem)<{ selected: boolean }>`
 		height: auto;
 		cursor: default;
 		overflow: hidden;
+		&:last-child, &:first-child {
+			border-radius: 0;
```

Comment:
what is this for?

Reply:
This is because when using the css method of sorting the last-child and first-child remain the same no matter which direction the sort is in. The result was that the border radius was being applied to the wrong items. 

I solved this by getting rid of the last-child first-child styling and instead add the border radius to the parent container with no overflow. I suppose I could make all dashboard lists use this method, although I'm cautious that this was initially done for a reason.